packages/vscode-extension/src/envTree.ts

```diff
@@ -88,7 +88,6 @@ export async function registerEnvTreeHandler(
             contextValue: contextValue,
             icon: provisionSucceeded ? "folder-active" : "symbol-folder",
             isCustom: false,
-            expanded: isLocal ? undefined : true,
```

Comment:
Will the env node be expanded automatically after it's provisioned?

Reply:
It will not expanded automatically but it will show expander icon, user can click the icon to view details

Reply:
Looks the current user experience might be changed for the env tree, and the subscription/resource group info might be missed by user. Is it by design to change this experience?

Reply:
Discussed and will only remove expander icon for spfx project